packages/astro/package.json

```diff
@@ -163,6 +163,7 @@
     "http-cache-semantics": "^4.1.1",
     "js-yaml": "^4.1.0",
     "kleur": "^4.1.5",
+    "linkedom": "^0.18.4",
```

Comment:
Dev dep? I think we have cheerio already too

Reply:
I just stole the test from the markdoc package. All the integrations use linkedom, but I can switch it to cheerio.

Reply:
OK, changed to use cheerio